app/utils.py

```diff
@@ -29,3 +33,61 @@ def parse_phone_number(phone_number):
         stripped = '1' + stripped
     stripped = '+' + stripped
     return stripped
+
+
+def parse_to_db(db, filename):
+    viewport_default = '39.861204,-75.310357|40.138932,-74.928582'
```

Comment:
I now understand why we give it two sets of coordinates. Is there anyway we can do this dynamically? e.g. it takes a city (like "Philadelphia" and generates a viewport?